apps/cli/tf/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "sunodo_cli_region" {
+  description = "The sunodo cli AWS region"
+  type        = string
+  sensitive   = true
+}
+
+variable "sunodo_cli_zone_id" {
+  description = "The sunodo cli zone ID"
```

Comment:
Maybe use `sunodo_cli_dns_zone_id` or `sunodo_cli_route53_zone_id`. I'm afraid zone ID is too generic and could be used as names for other resources in AWS. 

What do you think?

Reply:
Ok, I'm assuming the zone ID here points to a `sunodo.io` zone, and `cli.sunodo.io` is just a CNAME entry pointing to CloudFront's FQDN.

Agreed with removing the `_cli_`  part.